### ISSUE TITLE
agent target address fields do not allow variables

### ISSUE THREAD

**Issue by lucamrgs** [opencybersecurityalliance/cacao-roaster#11]
The definitions of the properties for the "address" property, used in the agent-targets of type HTTP API, ssh, linux, net-address, contain regexes that do not allow variables to be placed instead of actual URL/IP/Mac/vlan values.

Besides being impractical, this is also not in line with the specification, which defines all values for address keys to be list of strings. See e.g. sec. 7.8 of CACAO V2 spec:
"address (required) | dictionary | The key for each entry in the dictionary MUST be a stringthat uniquely identifies one or more address types. The key(s) MUST be one of the following values dname(domain name), ipv4, ipv6, l2mac, vlan, or url. The dictionary value associated with each key MUST be a listof string that contains the corresponding address(es) for that particular key type."

It would probably be the case that in the schemas, such properties are left as string, or else change the regex validation such that that CACAO variables strings are also allowed.

**Comment by dlemire60** [comment]
Good point, however I think this isn't really a Roaster issue but a CACAO issue. I'd suggest you post this to either or both of:

- https://github.com/oasis-open/cacao-json-schemas/issues
- https://github.com/oasis-tcs/cacao/issues
